ingestion/src/metadata/profiler/processor/core.py

```diff
@@ -451,7 +454,7 @@ def _prepare_column_metrics(self) -> List:
                 column=column,
                 table=self.table,
             )
-            for column in columns
+            for column in self._columns
```

Comment:
you should call the property attribute not the private one `self.columns` ln. 142

Reply:
Property attribute whenever called, executes multiple times, i think it is needed only ones per table, i have modified it to be used once, and store it in init and call it then